### PR TITLE
draft post - trumanitarian podcast

### PR DESCRIPTION
**Pull request by keyboardguy25**
### Describe the background of your pull request

added blog post draft for trumanitarian podcast episode with Neil

### Steps

- [x] Draft upload
- [ ] Review
- [ ] Publish

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.
 

## Summary by CodeRabbit

- **New Features**
  - Introduced a blog post featuring an interview with Neil Smyth, CEO of Alkemio, discussing the societal implications of profit-driven digital platforms.
  - Highlighted Alkemio's mission, commitment to collective action, and open-source approach.
  - Provided insights on the concept of "techno-feudalism" and a call to rethink digital tools.
  - Included a call to action to listen to the full podcast episode for deeper understanding.

### CODE REVIEW

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Neil addresses a critical issue in today’s digital landscape: profit-driven platforms control the very infrastructure of our online spaces. With the analogy of building a house but not owning the foundations, Neil illustrates that this isn't a hypothetical future; it’s our present reality. This phenomenon, often referred to as "techno-feudalism," is already here.
+
+### Introducing Alkemio: Collective Action Reimagined
+
+Alkemio was designed to enable collective action, providing safe digital spaces for collaborative work, backed by an organization dedicated to societal benefit.
+
+A significant milestone for Alkemio was its adoption of Steward Ownership. This commitment assures that Alkemio’s purpose remains central: the venture cannot be sold, and profit serves purpose. This diverges starkly from the prevailing model of digital platforms, where shareholder profits often take precedence over societal interests.
+
+Additionally, Alkemio’s Open Source commitment guarantees a transparent exit option for users. While Alkemio hopes to sustain its vision through a commercially hosted service, the open-source promise means users can leave if necessary.
```
Comment (rtharakan) [inline content/en/post/2024-11-trumanitarian-ep-92/index.md:47]: On the second aspect of open source (in addition to transparency), what are your thoughts on framing it not as users leaving, however, rather as complete freedom and ability for users to run their instance? The difference in this argument is that open source is not so much a freedom to exit, but a freedom to build and maintain.

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+The vision is for Alkemio to be a seed for a larger collaboration, creating a digital foundation with public interests at the core.
+
+### The Challenges of Scaling a Mission-Driven Platform
+
+Among the many themes discussed, Lars Peter asked Neil what’s needed to scale a platform like Alkemio. Competing with established tech giants means breaking free from their playbook.
```
Comment (rtharakan) [inline content/en/post/2024-11-trumanitarian-ep-92/index.md:53]: Suggestion: "Among the themes discussed was Alkemio's approach to scale and competing in a marketplace with established tech giants. Neil emphasized the need to break away from the existing Silicon Valley playbook and center our public values and interests to develop technology responsibly. 

It's a bit of a paraphrase - would like @techsmyth's approval before printing.

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Neil believes that only by offering a different narrative and fundamentally different product can a new digital foundation for society be realized. Attempting to compete within the shareholder-driven model simply won’t work.
```
Comment (rtharakan) [inline content/en/post/2024-11-trumanitarian-ep-92/index.md:55]: A suggestion: "At Alkemio, we believe that competing within the shareholder-driven model is playing a game whose rules are fundamentally antithetical to our vision of a society that benefits from technological progress. The time is overdue to amend the rules and reimagine what an alternative result could look like ."

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+However, scaling Alkemio still requires capital, though the model ensures this capital doesn’t bring voting rights or control. This safeguards the mission, while offering investors reasonable returns.
```
Comment (rtharakan) [inline content/en/post/2024-11-trumanitarian-ep-92/index.md:57]: Should we go into details of voting rights and control here? I'd suppose that leaving it a bit more high-level might be easier for a general audience. Thoughts? 

A suggestion: "However, scaling Alkemio still requires capital, though the organization model ensures this capital doesn't dilute our mission or veer from our public interest ambitions. "

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+Over time, the network effect of many communities engaging in collective action will drive the paradigm shift needed to embed this vision into society.
+
+### Rethinking Our Digital Tools
+
+One of the most compelling takeaways from the conversation is Neil’s challenge to listeners: reconsider the digital tools you use every day. Are they serving your interests, or are you serving theirs? And as AI becomes more integrated, have you thought about its effects on your autonomy?
+
+Neil encourages us to imagine a digital ecosystem where collaboration is free from profit-driven pressures.
```
Comment (rtharakan) [inline content/en/post/2024-11-trumanitarian-ep-92/index.md:65]: A minor suggestion: ".. where collaboration and building community is..".

**Review by rtharakan — COMMENTED** [review]
Leaving some comments for your thoughts. Happy to elaborate further.

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 1 above)
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:47]: Agreed, much more positive framing

File: content/en/post/2024-11-trumanitarian-ep-92/index.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+### The Problem with the Digital Status Quo
+
+Neil addresses a critical issue in today’s digital landscape: profit-driven platforms control the very infrastructure of our online spaces. With the analogy of building a house but not owning the foundations, Neil illustrates that this isn't a hypothetical future; it’s our present reality. This phenomenon, often referred to as "techno-feudalism," is already here.
+
+### Introducing Alkemio: Collective Action Reimagined
+
+Alkemio was designed to enable collective action, providing safe digital spaces for collaborative work, backed by an organization dedicated to societal benefit.
+
+A significant milestone for Alkemio was its adoption of Steward Ownership. This commitment assures that Alkemio’s purpose remains central: the venture cannot be sold, and profit serves purpose. This diverges starkly from the prevailing model of digital platforms, where shareholder profits often take precedence over societal interests.
```
[the remark this answers is not in the document]
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:45]: Good point

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 2 above)
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:53]: Needs maybe a bit more attention: key is that the reader grasps that we are playing a different game. That needs to standout...

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 3 above)
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:55]: Explain that to a 7 year old :) Words need to get a bit simpler (also my suggestions)

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 4 above)
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:57]: Not about dilution. Maybe better to phrase as keeping the purpose locked in and that value ultimately goes to the purpose?

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 5 above)
Comment (techsmyth) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md:65]: Maybe also replace collaboration with collective action and connecting with each other?

**Comment by techsmyth** [comment]
@rtharakan thanks for the comments. Feel free to make the edits as a PR so that we have the latest version...

**Comment by keyboardguy25** [comment]
> @rtharakan thanks for the comments. Feel free to make the edits as a PR so that we have the latest version...

@rtharakan if you want, we can go through the edits (and how to's if needed) together once you have your commits all set up! :)

**Comment by rtharakan** [comment]
> > @rtharakan thanks for the comments. Feel free to make the edits as a PR so that we have the latest version...
> 
> @rtharakan if you want, we can go through the edits (and how to's if needed) together once you have your commits all set up! :)

Thank you, @keyboardguy25 - and yes, please. I will revert after my session with Carlos and we can set-up a time.

File: content/en/post/2024-11-trumanitarian-ep-92/index.md (hunk 4 above)
Comment (keyboardguy25) [reply content/en/post/2024-11-trumanitarian-ep-92/index.md]: @rtharakan and I used the words "society-first" goal: please check if it's good wording in this section

**Comment by keyboardguy25** [comment]
@rtharakan and I went through these suggestions and changes together, most of these were implemented but there are still a couple of unresolved conversations that need to be looked at @techsmyth

**Review by techsmyth — APPROVED** [review]

**Pull request merged**